infra/modules/gcp-psoxy-bulk/variables.tf

```diff
@@ -52,6 +52,11 @@ variable "artifacts_bucket_name" {
   description = "Name of the bucket where artifacts are stored"
 }
 
+variable "artifact_repository_id" {
+  type        = string
+  description = "ID of the artifact repository"
+}
```

Comment:
provide `default = null` so anyone using this module directly won't break? (not aware of anyone using it directly in gcp case, but a couple are using aws directly)